src/components/ToolStylePopup/ToolStylePopup.js

```diff
@@ -112,6 +121,7 @@ class ToolStylePopup extends React.PureComponent {
     return (
       <div className={className} data-element="toolStylePopup" style={{ top, left }} ref={this.popup} onMouseDown={e => e.stopPropagation()} onClick={this.onClick}>
         <StylePopup
+          annotationType = {this.toolNameToAnnotationType(activeToolName)}
```

Comment:
It would be great if we can start a function name with a verb, what about `getAnnotationType` ?